tools/ci.psm1

```diff
@@ -177,27 +168,12 @@ function Invoke-AppVeyorFull
         Clear-PSRepo
     }
 
-    if($env:APPVEYOR)
-    {
-        throw "This function is to simulate appveyor, but not to be run from appveyor!"
-    }
-
-    if($APPVEYOR_SCHEDULED_BUILD)
-    {
-        $env:APPVEYOR_SCHEDULED_BUILD = 'True'
-    }
     try {
         Invoke-AppVeyorInstall
         Invoke-AppVeyorBuild
         Invoke-AppVeyorTest -ErrorAction Continue
         Invoke-AppveyorFinish
-    }
-    finally {
-        if($APPVEYOR_SCHEDULED_BUILD -and $env:APPVEYOR_SCHEDULED_BUILD)
-        {
-            Remove-Item env:APPVEYOR_SCHEDULED_BUILD
-        }
-    }
+    } finally {}
```

Comment:
Since the finally block is empty, we can just remove the try/finally.

Reply:
Done.